### ISSUE TITLE
When checking Accessibility using Wave Tool for mat-select that uses mat-label , getting Orphaned form label accessibility warning.

### ISSUE THREAD

**Issue by Viddhiyartha** [angular/material2#11686]
#### Bug, feature request, or proposal:
A drop down is created with mat-select and a label for dropdown is added with mat-label. On checking the accessibility using Wave Evaluation Tool chrome extension,  getting **Orphaned form label** Warning. (Label not associated with proper form element)

Wave Tool : https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh

#### What is the expected behavior?
Orphaned form label warning should not appear

#### What is the current behavior?
Orphaned form label warning apeard

#### What are the steps to reproduce?
Syntax to Reproduce the issue

![image](https://user-images.githubusercontent.com/32121160/41027535-e7d5c490-6994-11e8-8d53-76ded4cb6404.png)

#### What is the use-case or motivation for changing an existing behavior?
We are trying to fix accessibility issues in our project and we are relying on Material Designs.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular - 5.2.0 , Material - 5.2.3 , OS- Windows , TypeScript - 2.5.3 , Browser - Google Chrom

#### Is there anything else we should know?
Tried Adding id in mat-select /adding for in mat-label

![image](https://user-images.githubusercontent.com/32121160/41027344-64e1c17e-6994-11e8-8b0d-3521a7328a7b.png)

**Comment by crisbeto** [comment]
Closing since that warning doesn't get in the way of screen readers picking up and reading out the label.

**Comment by schejarla** [comment]
Hope, we don't see the issue, if we replace the label with span element.

![Alert Count After Span](https://user-images.githubusercontent.com/46920955/58563826-734a8d80-8249-11e9-8c9b-a6e1c562b869.PNG)
![Markup After Span](https://user-images.githubusercontent.com/46920955/58563828-734a8d80-8249-11e9-84e5-c864ef6b6d35.PNG)
![Markup Before Span](https://user-images.githubusercontent.com/46920955/58563831-73e32400-8249-11e9-8ea3-116df1e679fa.PNG)
![Alert at Element](https://user-images.githubusercontent.com/46920955/58563832-73e32400-8249-11e9-986c-43972882a125.PNG)
![Alert Count Before Span](https://user-images.githubusercontent.com/46920955/58563833-75145100-8249-11e9-9a89-a795e662f2db.PNG)
